source/includes/code-examples/Find.cs

```diff
@@ -0,0 +1,82 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+
+namespace CsharpExamples.UsageExamples;
+
+public class FindOne
+{
+    static IMongoCollection<Restaurant> _restaurantsCollection;
+    static string mongoConnectionString = "<Your Mongodb URI>";
+
+    public static void Main(string[] args)
+    {
+        setup();
+
+        // Find one document using builders
+        FindOneRestaurantBuilder();
+
+        // Extra space for console readability 
+        Console.WriteLine();
+
+        // Find one document using LINQ
+        FindOneRestaurantLINQ();
+    }
+
+    static void FindOneRestaurantBuilder()
+    {
+        // start-find-builders
+        var filter = Builders<Restaurant>.Filter
+            .Eq("name", "Bagels N Buns");
+
+        var restaurant = _restaurantsCollection.Find(filter).First();
+
+        Console.WriteLine(restaurant.ToBsonDocument());
+        // end-find-builders
+    }
+
+    static void FindOneRestaurantLINQ()
```

Comment:
Issue: Best practice to explicitly include access modifiers on class members. Public?

```suggestion
    public static void FindOneRestaurantLINQ()
```